### DISCUSSION TITLE
Tarde pero seguro - Practica CSS

### DISCUSSION THREAD

**Discussion by RIvanCF** [Launch-X-Latam/MisionFrontEnd#422] [Show and tell]
Se que estoy algo atrasado, pero me ando poniendo totalmente al corriente con mis practicas tanto frontend como backend. Así que les presento mi practica de CSS. 
Vaya que me costo mucho trabajo, pero al fin quedo. 
Me ayudarían mucho si le dan una estrellita 
### NOTA: NO ESTA ADAPTADO PARA SER VISUALIZADO EN UN DISPOSITIVO MOVIL, COMO LO HICE EN LA PRACTICA DE LA PASTELERIA, PURO DESKTOP 

<a href="https://rivancf.github.io/LaunchXLATAM_S3/"><img src="https://img.shields.io/badge/Practica%202-Vacunacion-orange" height="30"></a>
Enlace a la repo en la siguiente imagen:
<a href="https://github.com/RIvanCF/LaunchXLATAM_S3"><img src="https://img.shields.io/github/stars/RIvanCF/LaunchXLATAM_S3?style=social" height="30"></a>

También quiero aprovechar para que un equipo para el hackathon que le falte algún integrante, estoy disponible y comprometido con lo que se presente. También y espero admitan a otro mas.
Debemos seguirnos todos en GitHub y nuestras redes sociales para no perder contacto

**Comment by GonDev1** [comment]
Aun le falta algunos detalles que puedes mejorar desde el CSS pero es que esta buenazo este reto, más adelante seguro que lo vas a dominar...También aprovecho para decirte que estoy muy de acuerdo contigo en que debemos hacer Networking, la mayoría en lo que va del curso he visto que están navegando solos o se están juntando únicamente para los proyectos y no están aprovechando la red de contactos que se puede formar aquí, sin más quedo abierto para hacer equipo y de igual forma para agregarnos en nuestras redes, saludos ✌😉
